tests/test_edf.py

```diff
@@ -619,6 +619,11 @@ def test_read_edf_str():
     assert read_edf(str(EDF_FILE)).num_signals == 7
 
 
+def test_read_edf_expanduser():
+    with pytest.raises(FileNotFoundError, match="^[^~]*$"):
+        read_edf("~/test.edf")
```

Comment:
Maybe we should use a filename which is extremely unlikely to actually exist  :thinking:

Reply:
Yes exactly, I'm thinking about local execution. I'd use something like `edfio_testfile_that_should_not_exist.edf`. I agree, but the only other option I see is to use (os-dependent) environment variables to change what `~` expands to, so I'd rather go with this :D

Reply:
Well, a third option would be to not test this. I can see your point to make sure no one reverts this expansion in the future, but then I'd just fix it if it comes up.